lib/util/text_lines_iterator.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:flutter/widgets.dart';
+
+/// utililty class for traversing through multiline text
+class TextLinesIterator extends Iterator {
```

Comment:
I have no idea what this iterator does and what are its elements. This public `set current` scares me

Reply:
no need to fear :-)

Reply:
no need to fear :^)